src/azure-cli/azure/cli/command_modules/containerapp/_params.py

```diff
@@ -467,7 +468,7 @@ def load_arguments(self, _):
 
     with self.argument_context('containerapp job stop') as c:
         c.argument('job_execution_name', help='name of the specific job execution which needs to be stopped.')
-        c.argument('execution_name_list', help='comma separated list of job execution names.')
+        c.argument('execution_name_list', help='comma separated list of job execution names.', deprecate_info=Deprecated(self.cli_ctx, target='--execution_name_list', hide=True, message_func=lambda x: "Option 'execution_name_list' has been deprecated and will be removed in the next Cli version"))
```

Comment:
Please add test for deprecate argument to make sure when customer input it, the command will not throw error.